wiki.js

```diff
@@ -0,0 +1,56 @@
+//Module for getting 'clues' from wikipedia, EG flags or language
+
+// test stuff
+var country = {country: 'Germany'};
+
+
+
+/**
+* Uses the object from the flickr request to get information from wikipedia to
+* generate a request for the wikipedia information. Returns a promise.
+*/
+function getCountryClues(flickrCountry) {
```

Comment:
Looks like this parameter is actually the photo object. 

One step better: a more accurate parameter name.
Several steps better: just the country as a parameter.